invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides

```diff
@@ -97,8 +95,18 @@
       font-weight: normal;
 
     }
+
+    .dropdown.icon {
+      padding-top: 0.25em;
+    }
   }
 
+.members-icon-container {
+  width: 1em;
```

Comment:
I'm not sure what is the purpose of the width here.

Reply:
This is reserving space by creating a small container for the success / error icon. The dropdown is `fluid`, meaning that it will use all the space available. When an icon appears, that same space becomes smaller and the dropdown is re-scaled while the icon appears thus creating this issue.